apps/marketplace/pages/QuestionPage.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter, Switch, Route, BrowserRouter, Redirect } from 'react-router-dom'
+import QuestionsHeader from 'marketplace/components/Questions/QuestionsHeader'
+import SellerQuestions from 'marketplace/components/Questions/SellerQuestions'
+import PublishedAnswers from 'marketplace/components/Questions/PublishedAnswers'
+import { hasPermission } from 'marketplace/components/helpers'
+import { rootPath } from 'marketplace/routes'
+
+export class QuestionPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      brief: {
+        title: '',
+        internalReference: '',
+        id: ''
+      },
+      questionCount: {
+        questions: 0,
+        answers: 0
+      }
+    }
+  }
+
+  questionCountUpdated(questionCount) {
+    this.setState({
+      questionCount
+    })
+  }
+
+  briefUpdated(brief) {
+    this.setState({
+      brief
+    })
+  }
+
+  render() {
+    const { isPartOfTeam, isTeamLead, teams } = this.props
+    const briefId = this.props.match.params.briefId
+
+    if (!hasPermission(isPartOfTeam, isTeamLead, teams, 'answer_seller_questions')) {
+      return <Redirect to={`${rootPath}/request-access/answer_seller_questions`} />
+    }
+
+    return (
+      <BrowserRouter basename={`${rootPath}/brief/${briefId}/questions`}>
+        <div>
+          <QuestionsHeader brief={this.state.brief} questionCount={this.state.questionCount} />
```

Comment:
Can we use the `PageHeader` and `PageNavigation` components here?

Reply:
done